### PR TITLE
Bumped lodash version to 4.17.20 Closes #1070

### LINKED ISSUE #1070: Prototype Pollution by lodash's current version(4.17.15) that yup uses
**Describe the bug**
This package uses lodash package v4.17.15, and that version of lodash contains Prototype Pollution in `zipObjectDeep` method

**To Reproduce**
Bump lodash version to 4.17.20

### PR DESCRIPTION
**Pull request by agilatakishiyev**

### CODE REVIEW

**Comment by jquense** [comment]
this is handled by the lock file already. Happy to have it explicitly bumped in the package.json but don't include th `package-lock` file in the diff

**Review by Khadhiri786 — APPROVED** [review]

**Comment by agilatakishiyev** [comment]
> this is handled by the lock file already. Happy to have it explicitly bumped in the package.json but don't include th `package-lock` file in the diff

Ok, i'll remove it in the next commit

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.10.5",
-    "lodash": "^4.17.15",
+    "lodash": "4.17.20",
```
Comment (jquense) [inline package.json:103]:
```suggestion
    "lodash": "^4.17.20",
```

**Comment by jquense** [comment]
thanks!

**Pull request merged**